### PR TITLE
Add :fetch, :store methods for session & cookies hash

### PR DESCRIPTION
**Pull request by jelaniwoods**
Relating to this issue [#80](https://github.com/firstdraft/appdev/issues/80)

Adds initializer that makes `:fetch` and `:store` alias methods for `:[]` and `:[]=` for the `session` and `cookies` Hashes.

### CODE REVIEW

File: files/fetch_store_patch.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    end
+  end
+  
+  class Cookies
+    class CookieJar
+      alias_method :fetch, :[]
+      alias_method :store, :[]=
+    end
+  end
+end
```
Comment (raghubetina) [inline files/fetch_store_patch.rb]: There seems to be a little line ending warning here.

**Comment by raghubetina** [comment]
LGTM! I think it's worth making a card to eventually extract this into a gem so we can add tests.

File: files/fetch_store_patch.rb (hunk 1 above)
Comment (jelaniwoods) [reply files/fetch_store_patch.rb]: Good catch!

**Comment by LoganDSPrice** [comment]
This is great!

**Pull request merged**